applications/matter_bridge/README.rst

```diff
@@ -112,8 +141,13 @@ User interface
     :end-before: matter_door_lock_sample_led1_end
 
 Button 1:
-     If pressed for six seconds, it initiates the factory reset of the device.
-     Releasing the button within the six-second window cancels the factory reset procedure.
+    Depending on how long you press the button:
+
+    * If pressed for less than three seconds, it initiates the SMP server (Security Manager Protocol).
```

Comment:
```suggestion
    * If pressed for less than three seconds, it initiates the SMP (Security Manager Protocol) server.
```
Different SMP? It was Simple Management Protocol earlier.

Reply:
Wow I haven't noticed it before. Of course you are right. I basically did a copy-paste from existing docs in another sample, so it looks like we have this issue from a while. I will align another samples as well.